server/datastore/mysql/mdm.go

```diff
@@ -1318,9 +1318,7 @@ func (ds *Datastore) CleanSCEPRenewRefs(ctx context.Context, hostUUID string) er
 	stmt := `
 	UPDATE nano_cert_auth_associations
 	SET renew_command_uuid = NULL
-	WHERE id = ?
-	ORDER BY created_at desc
-	LIMIT 1`
+	WHERE id = ?`
```

Comment:
`ORDER BY ... LIMIT 1` is problematic in combination with the prior implementation of `GetHostMDMCheckinInfo` above.

Because new records are created with the `Authenticate` triggered by the renewal, we can't rely on the `created_at` timestamp to identify the prior record (i.e. the newly created record will be the one targeted, which is not what we want).

I haven't come up with any reason not to reset all renew command references for a given host uuid, but I'd appreciate others giving this some careful consideration as well.